src/erlcloud_aws.erl

```diff
@@ -375,9 +378,9 @@ http_headers_body({ok, {{Status, StatusLine}, _Headers, Body}}) ->
 http_headers_body({error, Reason}) ->
     {error, {socket_error, Reason}}.
 
-timeout(#aws_config{timeout = undefined}) ->
-    ?DEFAULT_TIMEOUT;
-timeout(#aws_config{timeout = Timeout}) ->
+get_timeout(#aws_config{timeout = undefined}) ->
```

Comment:
it's not simply renamed. it's now exported so that is can be used in other modules.
I find usage `erlcloud_aws:get_timeout(Cfg)` more informative, readable and matching function purpose then simple `erlcloud_aws:timeout(Cfg)`